### DISCUSSION TITLE
Build OnnxRuntime without Python

### DISCUSSION THREAD

**Discussion by drvshavva** [microsoft/onnxruntime#9526] [Other Q&A] [answered]
If there is no python in the operating system I use, can I build and use onnxruntime? Is there any way to do this?

**Comment by snnn** [comment, marked as answer]
I'm sorry, no.

**Reply by drvshavva** [reply]
Thank you for your answer

**Reply by drvshavva** [reply]
Why not ? As I see, in building process requires build.py, if I get rid off python script in build and somehow convert it to pure bash script can I build onnxruntime without python.

**Reply by snnn** [reply]
https://github.com/onnx/onnx/blob/master/CMakeLists.txt#L258
